tests/resources/example_docker_project/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM continuumio/miniconda3:4.6.14
+FROM continuumio/miniconda3:latest
```

Comment:
The tag `4.6.14` is old (last update was 2 years ago):

https://hub.docker.com/layers/continuumio/miniconda3/4.6.14/images/sha256-6b5cf97566c3b1d8bfd4ff1464fbdaaa9d9737c26d1b153eb3e88358d3826c48?context=explore